wordpress-mobile/WordPress-Android
# Use custom views for layouts that use vector drawables

Fixes #5560 by implementing a workaround that sets vector drawables programmatically. The upside is that this will work for all supported API's! The downside is that we need a custom view for each Android view that we use vector drawables for 😞 

To test:
* Run the app with API < 5.0 and >= 5.0
* Check around the activities who's views are affected (Stats, Gallery settings, post location, My Site, etc...)
* Verify that the icons with drawable attributes are showing correctly

cc @theck13 because of your previous work with these 😄
